### DISCUSSION TITLE
Do We Need Gitter or a Similar Chat Platform?

### DISCUSSION THREAD

**Discussion by xee5ch** [oscal-club/community#10] [Ideas]
Those who follow the OSCAL community know there is a [Gitter channel](https://gitter.im/usnistgov-OSCAL/Lobby). I would like some :+1:  or :-1: on this topic. If the need evolves, maybe we can grow our own chat platform organically. I would prefer to keep things asynchronous and easily searchable on the web, but I know people love to DM for more private stuff and that does have value too.

Let us know and provide feedback, thanks!

**Comment by jheadley** [comment]
The gitter channel seems popular and active to me. I'm fine with sticking to that. I too prefer asynchronous. Too many meetings during the day to keep up with direct messages.

**Reply by xee5ch** [reply]
I guess the question is whether we should make ourselves known in an official capacity there, but I am definitely cool with the status quo!

**Reply by jheadley** [reply]
I think that's your judgment call. I'm very new to the OSCAL conversation.
